xbmc/cores/AudioEngine/Engines/ActiveAE/ActiveAESound.h

```diff
@@ -59,6 +61,7 @@ class CActiveAESound : public IAESound
   static int64_t Seek(void *h, int64_t pos, int whence);
 
 protected:
+  CActiveAE *m_activeAE;
```

Comment:
@paxxi
> This adds to the mess we're trying to clean up and get away from.

WE dropped CAFactory and this dependency was already there. If we redesign it this PRS will get even bigger and worser to review. So please do not block this PR!